testing/yodl/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=yodl
+pkgver=3.08.02
+pkgrel=0
+pkgdesc="a pre-document language and tools to process it"
+url="https://fbb-git.github.io/yodl/"
+arch="all"
+license="GPL2"
+makedepends="icmake bash perl flex bison"
+subpackages="$pkgname-doc"
+source="yodl-$pkgver.tar.gz::https://github.com/fbb-git/yodl/archive/$pkgver.tar.gz"
+builddir="$srcdir/yodl-$pkgver/yodl"
+options="!check"
+
+prepare() {
+	cd "$builddir"
+	sed -i 's!DOCDOC\s*=.*$!DOCDOC = BASE + "/share/doc/yodl";!g' INSTALL.im
+}
+
+build() {
+	cd "$builddir"
+	for x in programs macros man ; do
+		./build $x
+	done
+}
+
+package() {
+	cd "$builddir"
+	for x in programs macros man ; do
```

Comment:
Variables used only in a single function should be declared as `local`, e.g. `local x; for x in …; do`. Also it’s better to use a bit more descriptive variable name, e.g. `name` in this case.